llvm/test/TableGen/invalid-type-cast-patfrags.td

```diff
@@ -19,21 +19,21 @@ def INSTR_BAR : Instruction {
 }
 
 #ifdef ERROR1
-// ERROR1: [[@LINE+1]]:1: error: {{.*}} Invalid number of type casts!
+// ERROR1: error: Invalid number of type casts!
```

Comment:
This looks like a regression, the test no longer prints a line number?

Reply:
And use the example input https://github.com/llvm/llvm-project/issues/157619 as a new unit test

Reply:
I was wondering if we can use PrintError instead or PrintFatalError, because ParseTreePattern does return nullptr even before my change. However for the test in the bug, FindPatternInputsAndOutputs then crashes because it doesn't handle the case when the children of Pat are nullptr.

Should I then also fix FindPatternInputsAndOutputs, so that we can use PrintError in ParseCastOperand ?

Reply:
That would probably be better